flint.ui/src/components/FileUpload/FileUpload.vue

```diff
@@ -1,54 +1,87 @@
 <template>
   <a-row :gutter="[24, 24]" class="pt-4 pb-6">
-    <a-col :md="8" :span="24">
-      <vue-dropzone
+    <a-col :md="12" :lg="8" :span="24">
+      <div v-if="files.config.length" class="text-2xl text-center mb-2">Config Files</div>
+      <div
+        v-if="!files.config.length"
         id="dropzoneConfig"
-        ref="myVueDropzoneConfig"
-        :options="dropzoneOptionsConfig"
-        :use-custom-slot="true"
-        @vdropzone-removed-file="fileRemoved"
-        @vdropzone-file-added="fileAddedConfig"
+        class="dropzone-gcbm"
+        :style="{ border: resultConfig.isDragActive.value ? '2px dashed #0087ff' : '2px dashed #bdbdbd' }"
+        ref="dropzoneConfig"
+        v-bind="resultConfig.getRootProps()"
       >
         <div class="dropzone-custom-content">
+          <input v-bind="resultConfig.getInputProps()" />
           <span class="font-semibold text-xl text-blueGray-700">Drag and drop to upload</span><br />
           <span>CONFIG FILES</span>
         </div>
-      </vue-dropzone>
+      </div>
+      <div v-else class="files-container">
+        <div v-for="(file, index) in files.config" :key="'file-gcbm-config-' + index" class="file-container">
+          <span class="text-lg text-blueGray-700">
+            {{ file.name }}
+          </span>
+          <br />
+          <span>{{ parseFloat((file.size / 1000).toFixed(2)) + ' KB' }}</span>
+        </div>
+      </div>
     </a-col>
 
-    <a-col :md="8" :span="24">
-      <vue-dropzone
+    <a-col :md="12" :lg="8" :span="24">
+      <div v-if="files.db.length" class="text-2xl text-center mb-2">Database</div>
+      <div
+        v-if="!files.db.length"
         id="dropzoneDB"
-        ref="myVueDropzoneDB"
-        :options="dropzoneOptionsDB"
-        :use-custom-slot="true"
-        @vdropzone-removed-file="fileRemoved"
-        @vdropzone-file-added="fileAddedDB"
+        class="dropzone-gcbm"
+        :style="{ border: resultDB.isDragActive.value ? '2px dashed #0087ff' : '2px dashed #bdbdbd' }"
```

Comment:
Can we replace the inline styling with design system styling or a css class for this case?